shardingsphere-agent/shardingsphere-agent-core/src/main/java/org/apache/shardingsphere/agent/core/plugin/AgentPluginLoader.java

```diff
@@ -117,10 +117,20 @@ public void loadAllPlugins() throws IOException {
                 log.info("Loaded jar {}", each.getName());
             }
         }
-        loadPluginDefinitionServices(ignoredPluginNames, pointMap);
+        Set<String> pluginNames = getPluginNames();
```

Comment:
Could Set<String> be changed to Collection<String>?

Reply:
Okay, I'll adjust it